### ISSUE TITLE
2.1.4 Captions - Core Requirement Suggestion

### ISSUE THREAD

**Issue by utkudyn-a11y** [w3c/wcag3#618]
A high contrast background is significant for partially-sighted persons for readibility of captions. So, at least 4,5:1 contrast ratio should be provided for caption text and its background.
This should be included in addition to captions adjustable supplemental requirement because it is directly affecting usability of people with low vision.
